CMakeLists.txt

```diff
@@ -10,23 +10,20 @@ endif(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.12.0")
 # Add our own cmake goodies to cmake module search path
 list(APPEND CMAKE_MODULE_PATH
      ${EKAT_CMAKE_PATH}
-     ${EKAT_CMAKE_PATH}/pkg_build
+     ${EKAT_CMAKE_PATH}/tpls
 )
 
-# Scripts containing macros needed to handle TPLs
-include(EkatBuildKokkos)
-include(EkatBuildYamlCpp)
-include(EkatBuildSpdlog)
-
 # EKAT has mostly C++, but (optionally) Fortran too
 option (EKAT_ENABLE_FORTRAN "Whether EKAT Fortran support is enabled." ON)
+message ("cm search path: ${CMAKE_PREFIX_PATH}")
```

Comment:
Do you want to keep messages like this one?

Reply:
Whoops!